lib/tinyset.rs

```diff
@@ -41,18 +47,26 @@ impl<'a, X:u64set::Fits64 + Ord + 'a> Iterator for Set64Iter<'a, X> {
     }
 }
 
-impl <'a, T: tinyset::Fits64 + Ord> tinyset_Set64<T> {
-    pub fn iter(&'a self) -> Set64Iter<'a, T> {
+impl <T: tinyset::Fits64 + Ord> tinyset_Set64<T> {
+    pub fn iter(&self) -> Set64Iter<T> {
         Set64Iter::new(self)
     }
 }
 
+impl <'a, T: tinyset::Fits64 + Ord> tinyset_Set64<T> {
```

Comment:
Why is lifetime parameter needed here? It compiles fine without it.

Reply:
Ok, removed the lifetime